CustomTabBar/CustomTabBar.swift

```diff
@@ -69,12 +69,24 @@ public protocol CustomTabBarDelegate {
         }
     }
     
+    public var titleFont: UIFont = UIFont.systemFontOfSize(16) {
```

Comment:
The difference is that we supported two default font sizes before. If you had an image, it defaulted the text size to 10, otherwise it had the default button size of 16. We should consider something like that instead, so that we add the possibility of overriding the font, but preserve the defaults. Maybe just use an optional?